baitap-submit/toidicodedao/01-prompt-engineering/01-prompt-engineering.md

```diff
@@ -1,11 +1,56 @@
 - Prompt 1
-
+  As a teching assistance, make a list of multiple choice question has at least 10 questions about a lesson. The lesson has content delimited by quote:
+  "
+  [Lesson Content]
+  "
 - Prompt 2
-
+  I have a short paragraph, analyze the meaning of the paragraph in the double quotes
+  ""
+  The character's true given name is never revealed to the reader. After having run away from home, he chooses the new name "Kafka", in honor of writer Franz Kafka. Kafka is described as being muscular for his age and a "cool, tall, fifteen-year-old boy lugging a backpack and a bunch of obsessions". He's also the son of the famous sculptor Koichi Tamura. His mother and sister left the family when he was four years old and he can't remember their faces. He occasionally interacts with his metaphysical alter ego "The boy named Crow" ("Kafka" sounds like "kavka", which means "jackdaw", a crow-like bird, in Czech). Crow tells Kafka throughout the novel that he must be "the toughest fifteen-year-old in the world" and thus motivates him to pursue the journey of running away from home. It is heavily suggested throughout the novel that he, Miss Saeki, and Nakata are somehow connected by an 'alternate reality' on which metaphysical objects from people's subconsciousness take form leading them to find an 'essence' to their lives in exchange for taking away a 'part' of their soul.
+  ""
 - Prompt 3
-
+  Review 1: "The restaurant is very good, i really like this, i will come back again." => positive review
+  Review 2: "very terrible, everything is bad, food has bad smell, it is the last time i come to this restaurant." => negative review
+  From a list reviews, please categorize the review in the list into positive and negative and count the total number of comments.
```

Comment:
Em cũng nên set role cho câu này. Và thêm ví dụ cho review k rõ ràng, ambiguity. Output format the nào